internal/model/yaml_test.go

```diff
@@ -14,9 +14,8 @@ func TestYAML_rxFilter(t *testing.T) {
 		e     fuzzy.Matches
 	}{
 		"empty-lines": {
-			q:     "foo",
-			lines: []string{},
-			e:     fuzzy.Matches{},
+			q: "foo",
+			e: fuzzy.Matches{},
```

Comment:
Same here this is the zero value for this type.

Reply:
I left this occurrence because the expected slice in this case is an empty slice, while the zero value for this type is a `nil` slice. The alternative would be to not preallocate the slice in the function with `make`, but that would be imho less effective. What do you think?

Reply:
Doh! I stand corrected. You are absolutely right Mohamed. Thank you!!